.github/workflows/assign-labels.yml

```diff
@@ -41,6 +41,10 @@ jobs:
             let fs = require('fs');
             const obj = JSON.parse(fs.readFileSync('./labels.json'));
             console.log(obj)
+            const existingAssignees = await github.rest.issues.listAssignees({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            });
```

Comment:
Get list of existing assignees and if there are existing assignees, do not assign anyone else.